docs/changelog.md

```diff
@@ -4,14 +4,23 @@ title: Changelog
 permalink: /changelog/
 ---
 
-# 50.0.0-SNAPSHOT (In Development)
+# 51.0.0-SNAPSHOT (In Development)
 
-* [Commits](https://github.com/mozilla-mobile/android-components/compare/v49.0.0...master)
-* [Milestone](https://github.com/mozilla-mobile/android-components/milestone/109?closed=1)
+* [Commits](https://github.com/mozilla-mobile/android-components/compare/v50.0.0...master)
+* [Milestone](https://github.com/mozilla-mobile/android-components/milestone/110?closed=1)
 * [Dependencies](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Dependencies.kt)
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/master/buildSrc/src/main/java/Config.kt)
 
+
+# 50.0.0-SNAPSHOT
+
+* [Commits](https://github.com/mozilla-mobile/android-components/compare/v49.0.1...v50.0.0)
```

Comment:
```suggestion
* [Commits](https://github.com/mozilla-mobile/android-components/compare/v49.0.0...v50.0.0)
```

This should be from since the minor changes should be included in the next major version.

Reply:
🤔  I thought `49.0.1` was the previous version

Reply:
`49.0.0` is the major version, which is what the `50.0.0` was based off since they are both from the same branch (master). With our minor releases that are made off a separate branch, we can incorrect commits when comparing the minor release to the major one.

Reply:
👍